app/retail/templates/mission.html

```diff
@@ -359,9 +395,11 @@ <h4 class="text-left text-uppercase">
             class="icon"
             src="{% static "v2/images/mission/interact/swiss_army.svg" %}"
           />
+          {% blocktrans %}
           <span class="icon-text">
             Pragmatic over dogmatism.
```

Comment:
Should we update this to `Pragmatism over dogmatism`?

Reply:
I think so.